src/TodoApp/TodoApp.csproj

```diff
@@ -6,6 +6,7 @@
     <TypeScriptCompileBlocked>true</TypeScriptCompileBlocked>
     <TypeScriptToolsVersion>latest</TypeScriptToolsVersion>
     <UserSecretsId>TodoApp</UserSecretsId>
+    <Nullable>enable</Nullable>
```

Comment:
This is in the `Directory.Build.props` file already - is there some IDE tooling or something that doesn't like that being there?

Reply:
Missed that